DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: PHES.ODM.Doc
 Type: Package
 Title: PHES-ODM-Doc
-Version: 0.1.0
+Version: 2.0.1
```

Comment:
This version is reflective of the package not the dictionary and should not be updated yet.

Reply:
Copy that. I will revert.

Reply:
It looks like project scripts work if we want to automate the inclusion of version and other information into the YAML. However, my sense is that other, simpler approaches should be implemented in the next update for Quarto.

 https://quarto.org/docs/projects/quarto-projects.html#project-scripts